spec/integration/data_stream/manifest.spec.yml

```diff
@@ -232,6 +232,15 @@ spec:
               type: object
               additionalProperties: false
               properties:
+                mode: 
+                  description: >
+                    Mode of the index, can be `standard`, `time_series` or `lookup`.
+                    The `lookup` mode is used for data streams that are used as lookup indices.
+                  type: string
+                  enum:
+                    - standard
+                    - time_series
+                    - lookup
```

Comment:
We have the other `index_mode` property as an indication to handle time series mode in a different way. Would it be possible to keep using this setting only?
```suggestion
```